File: Modul-2/Soal-4/PRAK204-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>
#define PHI 3.14

void main() {
    int r, t;
    float volume, luas, keliling;

    printf("Masukan Nilai Jari-Jari dan Tinggi Bejana: ");
    scanf("%d %d", &r, &t);

    volume = (float)(PHI*r*r*t);
    luas = (float)(2*PHI*r*(r+t));
    keliling = (float)(2*PHI*r);

    printf("Volume: %.2f\nLuas = %.2f\nKeliling = %.2f", volume, luas, keliling);


}

File: Modul-2/Soal-4/PRAK204-2410817110016-MuhammadAzriRahman.py
PHI = 3.14

total_input = []
print("Masukan Nilai jari- jari dan tinggi bejana: ")
while len(total_input) < 2:
     nilai = input("")
     total_input.extend(nilai.split())

r, t = total_input
r, t = map(int, (r, t))

volume = PHI*r*r*t
luas = PHI*r*(r+t)
keliling = 2*PHI*r

print(f"Volume: {volume:.2f}\nLuas: {luas:.2f}\nKeliling: {keliling:.2f}")

